CanalTP/NavitiaComponent
# Add cache

# Description

This PR add the possibility to cache all the call to navitia.

# Issue

BOT-3042

# How to test

Just upgrade your app (TrafficReport for now) with this branch in the vendor section, read the Readme.md file to update the autowire section and try it. By default, you should see (with the symfony debug bar) that the navitia calls are cached.
